### DISCUSSION TITLE
If the verdaccio private server has multiple domains, the <base href> in the index.html may be inconsistent with the visited domain

### DISCUSSION THREAD

**Discussion by cobolbaby** [verdaccio/verdaccio#4652] [Q&A]
**Your Environment**
  
 * **verdaccio version**: 5.30.3
 * **node version** [12.x.x, 14.x.x]: ?
 * **package manager**: [npm@7, pnpm@6, yarn@2]
 * **os**: linux
 * **platform**: docker

**Describe the bug**

If the verdaccio private server has multiple domain names, the `<base href>` in the index.html may be inconsistent with the visited domain, which may cause problems.

**To Reproduce**

https://github.com/cobolbaby/dockerize-and-ansible/tree/master/repomanager

**Expected behavior**

**Screenshots, server logs, package manager log**

![image](https://github.com/verdaccio/verdaccio/assets/11460848/0dca052b-29cf-4a0f-9fb0-c463a67a864e)

**Configuration File (cat ~/.config/verdaccio/config.yaml)**

**Environment information**

**Debugging output**

- `$ NODE_DEBUG=request verdaccio` display request calls (verdaccio <--> uplinks)
- `$ DEBUG=verdaccio* verdaccio` enable extreme verdaccio debug mode (verdaccio api)
- `$ npm -ddd` prints:
- `$ npm config get registry` prints:

**Contribute to Verdaccio**

- [ ] I'm willing to fix this bug 🥇

**Comment by cobolbaby** [comment]
@juanpicado I have revised my question, please check it again.

**Reply by juanpicado** [reply]
This topic has been discussed several times, pleases check other discussions first.

**Reply by cobolbaby** [reply]
After configuring the `VERDACCIO_PUBLIC_URL=https://npm.itc.inventec.net/`, when I access `https://npm.itc.inventec.net/`, the frontend resources load correctly.

**Reply by juanpicado** [reply]
That's exactly the answer, it is o the website, if is not easy to find please share feedback would be great to revisit that.

**Comment by xiuxiumomo** [comment]
set conf/config.yaml  -> web:  html_cache: false
